ros_buildfarm/wrapper/apt_get.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+
```

Comment:
Well we should add it to all the files. I just noticed for these files because I saw the full diff. We can merge this as is I've opened a new ticket #320